src/editor/editor.directive.ts

```diff
@@ -166,28 +167,43 @@ export class FroalaEditorDirective implements ControlValueAccessor {
     if (!this._opts.events) {
       this._opts.events = {};
     }
+
+    // Store user event callbacks if exist
+    if (this._opts.events[eventName]) {
+      this._userEventCallbacks[eventName] = this._opts.events[eventName];
+    }
+
     this._opts.events[eventName] = callback;
   }
 
+  private executeUserCallback(eventName) {
+    if (this._userEventCallbacks[eventName]) {
+      this._userEventCallbacks[eventName]();
+    }
+  }
+
   private initListeners() {
     let self = this;
 
     // bind contentChange and keyup event to froalaModel
-    this.registerEvent(this._element, "froalaEditor.contentChanged", function() {
+    this.registerEvent(this._element, "contentChanged", function() {
       setTimeout(function() {
         self.updateModel();
+        self.executeUserCallback("contentChanged");
       }, 0);
     });
-    this.registerEvent(this._element, "froalaEditor.mousedown", function() {
+    this.registerEvent(this._element, "mousedown", function() {
       setTimeout(function() {
         self.onTouched();
+        self.executeUserCallback("mousedown");
```

Comment:
The callback should be executed right away, without having to call another method.

Reply:
@stefanneculai Done